hazelcast/src/main/java/com/hazelcast/monitor/impl/GlobalIndexesStats.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.monitor.impl;
+
+import java.util.concurrent.atomic.AtomicLongFieldUpdater;
+
+import static java.util.concurrent.atomic.AtomicLongFieldUpdater.newUpdater;
+
+/**
+ * The implementation of internal indexes stats specialized for global indexes.
+ * <p>
+ * The main trait of the implementation is the concurrency support, which is
+ * required for global indexes because they are shared among partitions.
+ */
+public class GlobalIndexesStats implements InternalIndexesStats {
```

Comment:
Names are seemed confusing. We have `GlobalIndexesStats` and `GlobalIndexStats`.

Reply:
Not sure what are better options. We already have `com.hazelcast.query.impl.Indexes` and `com.hazelcast.query.impl.Index`, their stats specialized for global indexes are named accordingly.

Reply:
seems ok.

Reply:
fixed